lib/terraforming/template/tf/sqs.erb

```diff
@@ -0,0 +1,11 @@
+<% queues.each do |queue| -%>
+resource "aws_sqs_queue" "<%= module_name_of(queue) %>" {
+    name                       = "<%= module_name_of(queue) %>"
+    visibility_timeout_seconds = <%= queue["VisibilityTimeout"] %>
+    message_retention_seconds  = <%= queue["MessageRetentionPeriod"] %>
+    max_message_size           = <%= queue["MaximumMessageSize"] %>
+    delay_seconds              = <%= queue["DelaySeconds"] %>
+    receive_wait_time_seconds  = <%= queue["ReceiveMessageWaitTimeSeconds"] %>
+}
```

Comment:
sqs Not included  `polixy` 
https://www.terraform.io/docs/providers/aws/r/sqs_queue.html#policy